data/4_to_be_checked/need-ms-parts/419.xml

```diff
@@ -369,6 +372,104 @@
               </bibl>
             </listBibl>
           </additional>
+          </msPart>
+          <msPart n="2" xml:id="Part2">
+            <msIdentifier>
+              <idno type="URI">http://syriaca.org/manuscript/1454</idno>
+              <altIdentifier>
+                <idno type="BL-Shelfmark">Add MS 14651 lower text</idno>
+              </altIdentifier>
+              <altIdentifier>
+                <collection>William Wright, Catalogue of the Syriac Manuscripts in the British Museum Acquired since the Year 1838</collection>
+                <idno type="Wright-BL-Arabic">948</idno>
+              </altIdentifier>
+              <altIdentifier>
+                <collection>William Wright, Catalogue of the Syriac Manuscripts in the British Museum Acquired since the Year 1838</collection>
+                <idno type="Wright-BL-Roman">DCCCCXLVIII</idno>
+              </altIdentifier>
+            </msIdentifier>
+            <msContents>
+              <summary/>
+              <textLang mainLang="syr">Syriac</textLang>
+              <msItem xml:id="p2a1" n="1" defective="true">
+                <locus from="1" to="217"/>
+                <title xml:lang="en">A manuscript of the Gospels, according to the Peshitta version</title>
```

Comment:
You can delete "A manuscript of" and just have "The Gospels, according to the Pehsitta version" as the title here.